sequence/create_sequence.m

```diff
@@ -1,22 +1,17 @@
 function [sequence] = create_sequence(directory, varargin)
-% create_sequence Create a new sequence descriptor
```

Comment:
Was the comment removed by mistake? Can you add it back, because the documentation is generated from these comments.